internal/mockexporter/loadtest.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
```

Comment:
I would not place this under internal. Usually we do not place there main packages.

When we have these we place them in `./cmd/DifferentCMD`. Should we move these files in a new folder there?

Reply:
I agree that `internal` is not the best place for it, however I'm also not sure of whether this is useful enough externally to give it its own subdir in cmd. I don't really have a preference for neither option. What are @gsanchezgavier and @arvdias thoughts on this?

Reply:
I would put it under /tools and if we do the automated load test move it to /test imho

Reply:
One of the CMD I believe it is used already for debugging only, but yes tools or test work as well 😃

Reply:
I moved this to `internal/cmd` since it already existed. I have zero problems moving it into `/tools` if you think it's a better fit:
🚀 : Move to `tools`
👍 : Keep in `internal/cmd`